### PR TITLE
fix(components): [table-body] keep row data reactive

### PR DESCRIPTION
**Pull request by jianjunyuu**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

fix the issure https://github.com/element-plus/element-plus/issues/8581

### CODE REVIEW

**Comment by chenxch** [comment]
@jianjunyuu Need to divide by the unused.

**Comment by jianjunyuu** [comment]
> @jianjunyuu Need to divide by the unused.

hi, i removed unused onUpdated.
please look again~

**Review by chenxch — APPROVED** [review]

**Comment by chenxch** [comment]
cc @element-plus/backers  Need to review again to confirm.

File: packages/components/table/src/table-body/index.ts [hunk 1]
```diff
@@ -52,9 +52,6 @@ export default defineComponent({
     onUnmounted(() => {
       removePopper?.()
     })
-    onUpdated(() => {
-      removePopper?.()
-    })
```
[the remark this answers is not in the document]
Comment (jianjunyuu) [reply packages/components/table/src/table-body/index.ts:54]: yes, When the reactive data is modified, onUpdated is triggered and the tooltip is removed.

**Review by tolking — APPROVED** [review]

**Pull request merged**